lib/mumuki/laboratory/mumukit/randomizations/range.rb

```diff
@@ -0,0 +1,6 @@
+class Mumukit::Randomizer::Randomization::Range < Mumukit::Randomizer::Randomization::OneOf
+
+  def initialize(lower_bound, upper_bound)
+    @choices = (lower_bound..upper_bound).to_a
```

Comment:
Ok, It is true that you can make it work this way, but I think it is weird that Rande inherits from OneOf - they have different semantics.

Reply:
I think it would be lass hacky and more intention revealing if you implement Range from scratch: 

````
def initialize(lower_bound, upper_bound)
  @lower_bound = ....
  @uper_bound ...
end

def get(value)
  ...
end
```

Reply:
If there is duplicated logic, I'd use a template method.

Reply:
Also, I'd suggest not using a_range.to_a since this means building the whole range of elements, which can be large and memory consuming. Instead, you could use - both on oneof and range - a `Random`:

```ruby
Random.new(seed).rand(0..(choices.size - 1)) 
Random.new(seed).rand(lower_bound..upper_bound) 
```